pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -161,17 +169,31 @@ type volumeBinder struct {
 	bindTimeout time.Duration
 
 	translator InTreeToCSITranslator
+
+	capacityCheck *CapacityCheck
```

Comment:
Eventually when this goes GA, I think this structure should be folded into the main struct

Reply:
I believe @cofyc has a fix for this in https://github.com/kubernetes/kubernetes/pull/92684